src/Microsoft.Health.Fhir.Core/Features/Operations/Export/ExportJobWorker.cs

```diff
@@ -37,6 +39,8 @@ public ExportJobWorker(Func<IScoped<IFhirOperationDataStore>> fhirOperationDataS
             _exportJobConfiguration = exportJobConfiguration.Value;
             _exportJobTaskFactory = exportJobTaskFactory;
             _logger = logger;
+
+            _delayBeforeNextPoll = _exportJobConfiguration.JobPollingFrequency;
```

Comment:
nit: I think move the declaration of `_delayBeforeNextPoll` into `ExecuteAsync` since its only mutating state within that function

Reply:
Moved.